src/App.js

```diff
@@ -14,11 +14,67 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    activeElement: null,
+    selectedGood: '-',
+  };
+
+  removeActiveClass = () => {
+    const { activeElement } = this.state;
+
+    if (activeElement) {
+      activeElement.classList.remove('active');
+    }
+  };
+
+  selectedGoodHandler = (e, good) => {
+    this.removeActiveClass();
+
+    e.target.classList.add('active');
+
+    this.setState({
+      activeElement: e.target,
+      selectedGood: good,
+    });
+  };
+
+  clearGoodsListHandler = () => {
+    this.setState({
+      selectedGood: '-',
+    });
+
+    this.removeActiveClass();
+  }
+
+  render() {
+    const { selectedGood } = this.state;
+
+    return (
+      <div className="App">
+        <button
+          onClick={this.clearGoodsListHandler}
+          className="clear"
+          type="button"
+        >
+          Clear
```

Comment:
<img width="468" alt="image" src="https://user-images.githubusercontent.com/38065883/92138856-9df59f80-ee17-11ea-9148-406787424fcf.png">
I think it would be logical, if this button appears if only there are selected good currently in state